android/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id("com.android.application")
+    kotlin("android")
+}
+
+android {
+    namespace = "vn.luongvo.kmm.survey.android"
+    compileSdk = 32
+    defaultConfig {
+        applicationId = "vn.luongvo.kmm.survey.android"
+        minSdk = 24
```

Comment:
Please use `buildSrc + Kotlin DSL` to define these values

Reply:
@suho Yes, I will update it in the next task https://github.com/luongvo/kmm-survey/issues/2 👌